head.md

```diff
@@ -1,21 +1,21 @@
-# \<oscd-component>
+# \<oscd-action-icon>
 
 This webcomponent follows the [open-wc](https://github.com/open-wc/open-wc) recommendation.
 
 ## Installation
 
 ```bash
-npm i oscd-component
+npm i @openscd/oscd-action-icon
 ```
 
 ## Usage
 
 ```html
 <script type="module">
```

Comment:
Can you add this line to the head.md?

### Icon font
Material Icons are being used for the icons. This font needs to be added in the html first.
You can add it like so
```html
<link href="https://fonts.googleapis.com/css?family=Material+Icons&display=block" rel="stylesheet">
```

Reply:
Done